Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -957,4 +1011,13 @@
   categories: [ HideSpawnMenu ]
   components:
   - type: Sprite
-    state: brown-1
\ No newline at end of file
+    state: brown-1
+
+- type: entity
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 0

(indentation)